main.go

```diff
@@ -241,6 +245,44 @@ func visit(url *url.URL) {
 	}
 }
 
+// readResponseBody spools the http response body to disk or to screen depending
+// on the value of SOME FLAG.
+func readResponseBody(response io.Reader) string {
+	var err error
+	if showBodyEnv := os.Getenv("HTTPSTAT_SHOW_BODY"); showBodyEnv != "" {
```

Comment:
it should be possible to rewrite this as
```go
showBody := getEnvOrDefault(showBodyEnv, showBody)
```
If `getEnvOrDefault` is written properly.